mmhuman3d/models/body_models/smplx.py

```diff
@@ -0,0 +1,247 @@
+from typing import Optional
+
+import numpy as np
+import torch
+from smplx import SMPLX as _SMPLX
+from smplx.lbs import vertices2joints
+
+from mmhuman3d.core.conventions.keypoints_mapping import (
+    convert_kps,
+    get_keypoint_num,
+)
+from mmhuman3d.core.conventions.segmentation import body_segmentation
+from ..builder import BODY_MODELS
+
+
+@BODY_MODELS.register_module(name=['SMPLX', 'smplx'])
+class SMPLX(_SMPLX):
+    """Extension of the official SMPL-X implementation."""
+
+    body_pose_keys = {'global_orient', 'body_pose'}
+    full_pose_keys = {
+        'global_orient', 'body_pose', 'left_hand_pose', 'right_hand_pose',
+        'jaw_pose', 'leye_pose', 'reye_pose'
+    }
+    NUM_VERTS = 10475
+    NUM_FACES = 20908
+
+    def __init__(self,
+                 *args,
+                 keypoint_src: str = 'smplx',
+                 keypoint_dst: str = 'human_data',
+                 keypoint_approximate: bool = False,
+                 joints_regressor: str = None,
+                 extra_joints_regressor: str = None,
+                 **kwargs):
+        """
```

Comment:
If type is required in docstring, the tool(VSCODE) below could generate docstring according to args' name and type.
https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring

Reply:
Refer to issue: #61